packages/superset-ui-legacy-plugin-chart-table/src/ReactDataTable.tsx

```diff
@@ -35,19 +36,9 @@ if (!dt.$) {
   dt(window, $);
 }
 
+const RE_HTML_TAG = /<.*>/; // a dead simple regexp to find html tag
```

Comment:
Let's rename this to `htmlTagRegex` for clarity

Reply:
Can make small adjustment to the regex.

`/(<([^>]+)>)/gi`

Reply:
I don't think we need the `gi` flags and captured groups. Will use `/<[^>]+>/`.

Reply:
obligatory: https://stackoverflow.com/a/1732454

Reply:
Lol. We didn't really use this to extract or remove the tags, just preemptively test whether some cell text is *potentially* HTML so we can avoid using `dangerouslySetInnerHTML`.